dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -157,7 +157,7 @@ start-task [ -w "172.16.30.15:10081"] ./task.yaml
 query task's status
 
 Usage:
- dmctl query-status [-w worker ...] [task_name] [flags]
+ dmctl query-status [-w worker ...] [task-name] [flags]
```

Comment:
Currently the output is `dmctl query-status [-w worker ...] [task_name] [flags]` from `help query-status`, will update dmctl output later?

Reply:
Do you think which of the following two is better:
1. keep them as the current output of dmctl
2. change them and update dmctl later

Reply:
I prefer 2, as we have already used `-` in command, we can keep them same

Reply:
I fire a PR in DM, https://github.com/pingcap/dm/pull/291.